apps/earn-protocol/app/server-handlers/tables-data/top-depositors/updater.ts

```
@@ -0,0 +1,94 @@
+      timestamp: Number(rate.timestamp),
+      fleetAddress: rate.fleetAddress,
+    })
+
+    return acc
+  }, {})
+
+  const latestRatesByFleet = Object.entries(ratesByFleet).reduce<{
+    [key: string]: FleetRate
+  }>(
+    (acc, [fleetId, _rates]) => ({
+      ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)